src/staff_augmentation/components/GetFreeQuote/GetFreeQuoteConfirmationMessage/styles.module.scss

```
@@ -0,0 +1,39 @@
+@import '/src/styles/theme.scss';
+@import '/src/styles/media_queries.scss';
+
+.confirmationBadge {
+  display: flex;
+  width: 252px;
+  height: 252px;
+  border-radius: 360px;
```

Using a percentage in this case is a better alternative in this case. If we want to change the size of the circle, it'll remain a circle.
```suggestion
  border-radius: 50%;
```